tests/test_port_dpb.py

```diff
@@ -0,0 +1,175 @@
+from swsscommon import swsscommon
+import redis
+import time
+import os
+import pytest
+from pytest import *
+import json
+import re
+from port_dpb import Port
+
+@pytest.yield_fixture(scope="class")
```

Comment:
Not sure if this should be "yield_fixture" or just a normal function.  Same for remove_dpb_config_file